index.html

```diff
@@ -207,16 +207,6 @@ <h2>
             method is called, the user agent MUST run the following
             steps:</p>
             <ol>
-              <li>
```

Comment:
Note that the diff-tool just thinks this was removed. In effect, I just put something ahead of it and then fixed the indentation.

Reply:
@jan-ivar

Reply:
I think the mental model for the developer should be kept as simple as possible. "I can call setFocusBehavior" all the way up to when gDM is called, then exactly one more time, at latest when gDM's promise resolves" - this is already at the limit. I don't want to add the caveat of "...unless gDM fails, in which case I can call it again." As a developer, if your gDM call fails, try again from scratch.

Reply:
I think the mental model for the developer should be kept as simple as possible. `"I can call setFocusBehavior() multiple times before gDM is called, then exactly one more time, at latest when gDM's promise resolves"` - this is already at the limit. I don't want to add the caveat of `"...unless gDM fails, in which case I can call it again."` 

As a developer, if your gDM call fails, try again from scratch.

It's a shame that failures over WebIDL bindings don't get captured here, but I see it as an acceptable edge-case to avoid worrying about. I don't think programs would typically run into this and then try to resolve such an error on the fly, programmatically. Rather, it will be a bug that someone will manually fix.